tests/views/test_message.py

```diff
@@ -1612,6 +1579,118 @@ def test_conversation_reply_no_edit_permission(self, mock_request, mock_get_jwt)
         self.assertNotIn("Close conversation".encode(), response.data)
         self.assertNotIn("Send message".encode(), response.data)
 
+    @requests_mock.mock()
+    @patch("response_operations_ui.controllers.message_controllers._get_jwt")
+    @patch("flask_login.utils._get_user")
+    def test_flash_redirect_to_technical_inbox(self, mock_request, current_user, mock_get_jwt):
+        self._client_session_("technical")
+        mock_get_jwt.return_value = "mock_jwt"
+        current_user.return_value.id = 1
+        # Post message on reply
+        thread_json["category"] = "TECHNICAL"
+        mock_request.get(url_get_thread, json=thread_json)
+        mock_request.post(url_send_message, json=threads_no_unread_list, status_code=201)
+
+        # Conversation list
+        mock_request.get(shortname_url + "/ASHE", json=ashe_info["survey"])
+        mock_request.get(url_get_threads_list, json=thread_list)
+        mock_request.get(url_get_surveys_list, json=survey_list)
+        mock_request.get(url_messages + "/count", json={"total": 1}, status_code=200)
+        mock_request.get(url_permission_url, json=user_permission_admin_json, status_code=200)
+        mock_request.post(url_sign_in_data, json={"access_token": self.access_token}, status_code=201)
+        response = self.client.post(
+            "/messages/threads/fb0e79bd-e132-4f4f-a7fd-5e8c6b41b9af", data=self.message_form, follow_redirects=True
+        )
+        self.assertIn("Message sent.".encode(), response.data)
+        self.assertIn("Messages".encode(), response.data)
+        self.assertIn("Technical Messages".encode(), response.data)
+
+    @requests_mock.mock()
+    @patch("response_operations_ui.controllers.message_controllers._get_jwt")
+    @patch("flask_login.utils._get_user")
+    def test_flash_redirect_to_misc_inbox(self, mock_request, current_user, mock_get_jwt):
+        self._client_session_("misc")
+        mock_get_jwt.return_value = "mock_jwt"
+        current_user.return_value.id = 1
+        # Post message on reply
+        thread_json["category"] = "MISC"
+        mock_request.get(url_get_thread, json=thread_json)
+        mock_request.post(url_send_message, json=threads_no_unread_list, status_code=201)
+
+        # Conversation list
+        mock_request.get(url_get_threads_list, json=thread_list)
+        mock_request.get(url_get_surveys_list, json=survey_list)
+        mock_request.get(url_messages + "/count", json={"total": 1}, status_code=200)
+        mock_request.post(url_sign_in_data, json={"access_token": self.access_token}, status_code=201)
+        response = self.client.post(
+            "/messages/threads/fb0e79bd-e132-4f4f-a7fd-5e8c6b41b9af", data=self.message_form, follow_redirects=True
+        )
+        self.assertIn("Message sent.".encode(), response.data)
+        self.assertIn("Messages".encode(), response.data)
+        self.assertIn("RFT Messages".encode(), response.data)
```

Comment:
Is that what the inbox is called RFT?

Reply:
One of these tests in each inbox needs to assert that a link is present, to return to the message

Reply:
Yeah it's called the "RFT Messages" inbox.

I've updated the tests.